akvo/rsr/migrations/0113_auto_20171110_1042.py

```diff
@@ -0,0 +1,171 @@
ion'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', 'application'), ('application', '
```

Comment:
Here, and in several other places the choices list is based on the buggy codelist_choices function that's fixed in #2966. 

I don't think this matters, it seems the choices list here is more of an annotation that the real thing since we're generating the choices dynamically via the function call and I can't see the list in the migration ever being used. Just wanna get a second opinion from you.

Reply:
You are right about the values in the migration not being used, but when this branch gets merged with develop (and the fixed `codelist_choices`), django will ask us to migrate, anyway - a new migration will become possible.  Instead, I just rebased this PR against the current develop, and fixed the migration file.